evalml/tests/component_tests/test_estimators.py

```diff
@@ -114,6 +116,8 @@ def test_estimator_predict_output_type(X_y_binary, helper_functions):
                        (X_df_with_col_names, y_series_with_name, X_df_with_col_names.columns, y_series_with_name.unique())]
 
     for component_class in _all_estimators_used_in_search():
```

Comment:
I think we should add Arima to the list of components not allowed in AutoMLSearch for the time being!